aloha-core/src/main/scala/com/eharmony/aloha/dataset/csv/CsvRowCreator.scala

```diff
@@ -92,7 +91,10 @@ final object CsvRowCreator {
                                 case EncodingBasedFinalizer(fnl) => fnl(encoding)
                             }
 
-                            val strFunc = success.andThenGenAggFunc(finalizer)
+                            val strFunc = success.
+                              andThenGenAggFunc(_ orElse csvCol.defVal).  // Issue #98: Inject defVal on None.
```

Comment:
This line is the one that allows the default to be returned when the function returns `None`.